src/pages/molecules/itemCartBtn/ItemCartBtns.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+// helper function
+import { incrementItemHandler, incrementCartHandler, decrementItemHandler, decrementCartHandler } 
+    from './helper/helperFunctions';
+
+class ItemCartBtns extends Component {
+    render() {
+        const { itemIndex, vegetableItem, setState } = this.props;
+        return (
+            <div className="billItemBtn">
+                <button className='decreaseItem' onClick={() => { 
+                    decrementItemHandler(itemIndex, setState); 
+                    decrementCartHandler(vegetableItem, setState); 
+                }}>-</button>
+                <div className='itemValue'>{vegetableItem.count}</div>
+                <button className='increaseItem' onClick={() => { 
```

Comment:
These could be further divided into separate molecules, so that we do not have to contain logic of increase and decrease, the respective component will take care of it.